mkosi/config.py

```diff
@@ -2891,6 +2894,20 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         parse=config_make_list_parser(delimiter=","),
         help="When building a kernel modules initrd, exclude the specified kernel modules",
     ),
+    ConfigSetting(
+        dest="firmware_include",
+        metavar="REGEX",
```

Comment:
Should we go for globs? I've long since realized that making the kernel modules include/exclude settings regex was a mistake.

Reply:
While that would be easier, having this disparate doesn't seem like a good idea, they should behave the same. We could break compat again and make the others globs as well.

Reply:
Well, I just kept what it was done for kernel modules...

Reply:
Let's just keep it as regex, I don't want to break compat on this since it's likely actually used often